modules/govuk_postgresql/manifests/server/primary.pp

```diff
@@ -42,12 +42,17 @@
       value => 'hot_standby';
     'max_wal_senders':
       value => 3;
-    'checkpoint_segments':
-      value => 8;
     'wal_keep_segments':
       value => 256;
   }
 
+  if $::postgresql::globals::version < '9.5' {
```

Comment:
We'll eventually have to convert this to a https://docs.puppet.com/puppet/latest/reference/function.html#versioncmp for when we move to puppet 4. I'm fine leaving it for later though.